### PR TITLE
added my profile

### PR DESCRIPTION
**Pull request by Metal-Milonga**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -134,12 +134,21 @@ a:visited {
   height: 400px;
 }
 
+<<<<<<< Updated upstream
```
Comment (bhollan) [inline css/styles.css]: These kinds of marks are the result of a merge conflict and should never be a part of a commit into any repo.

This is piece A.

The original file's contents are between piece A and piece B.

File: css/styles.css [hunk 2]
```diff
@@ -134,12 +134,21 @@ a:visited {
...
 #teresa-grosch-cover {
   background-image: url("http://imgur.com/VcbIKew.jpg");
 }
 
 #teresa-grosch-card {
   background-image: url("http://imgur.com/RWhByb2.jpg");
+=======
```
Comment (bhollan) [inline css/styles.css]: This is piece B

File: css/styles.css [hunk 3]
```diff
@@ -134,12 +134,21 @@ a:visited {
...
+#david-holton-cover {
+  background-image: url("http://i.imgur.com/Cilefxv.jpg?1");
+}
+
+#david-holton-card {
+  background-image: url("http://i.imgur.com/Cilefxv.jpg?1");
+>>>>>>> Stashed changes
```
Comment (bhollan) [inline css/styles.css]: This is piece C.

Your changes are usually between piece B and piece C.
Your changes look good, but you're overwriting another student's CSS rule.

File: index.html [hunk 4]
```diff
@@ -25,6 +25,19 @@
...
+                <h3 class="view-profile-text">View Profile</h3>
+              </div>
+              <div class="card-text-container">
+                <h4 class="student-name">David Holton</h4>
+                <p class="student-location">St Petersburg, FL</p>
+              </div>
+            </a>
+          </div>
+
+        <div class="roster-cards-container">
```
Comment (bhollan) [inline index.html]: This is an additional, erroneous `roster-cards-container` div and should be deleted.

**Comment by bhollan** [comment]
Hi @Metal-Milonga !

Looking through your changes, overall they look good.  I've made a few comments for a few changes that need to be made before we can merge your pull request.

As always, please feel free to Ask A Question and we'll help you through any difficulties you're having!

CC: @aturkewi

**Comment by aturkewi** [comment]
Hello! We recently made a change to this repo that caused all open PRs to be close and we are not able to re-open them. I'm very sorry about the inconvenience, but I just wanted to let you know why this PR was close and that we still welcome your profile on the site! So if you're still interested in having your profile up, please feel free to open another PR.

**Pull request closed without merging**